resources/views/product/partials/options/field.blade.php

```diff
@@ -1,8 +1,7 @@
-<label>
-    <x-rapidez::label>{{ $option->title }} {{ $option->price_label }}</x-rapidez::label>
-    <x-rapidez::input
-        :required="$option->is_require"
-        :maxlength="$option->max_characters ?: false"
-        v-model="addToCart.customOptions[{{ $option->option_id }}]"
-    />
-</label>
+<x-rapidez::input
+    :required="$option->is_require"
+    :maxlength="$option->max_characters ?: false"
+    v-model="addToCart.customOptions[{{ $option->option_id }}]"
+>
+    <x-slot:label>{{ $option->title }} {{ $option->price_label }}</x-slot:label>
```

Comment:
Attributes on this slot doesn't have effect? When I do `<x-slot:label class="text-red-500">`